test/e2e/mongodbtests/mongodbtests.go

```diff
@@ -24,6 +24,15 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
+// SkipTestIfLocal skips tests locally which tests connectivity to mongodb pods
+func SkipTestIfLocal(t *testing.T, msg string, f func(t *testing.T)) {
```

Comment:
as discussed on a call, `Skip()` will actually terminate the entire test. We can use

```golang
// SkipTestIfLocal skips tests locally which tests connectivity to mongodb pods
func SkipTestIfLocal(t *testing.T, msg string, f func(t *testing.T)) {
	if testing.Short() {
		t.Log("Skipping [" + msg + "]")
		return
	}
	t.Run(msg, f)
}

```

to achieve the desired behaviour